### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by nikita-ezhkov**
Some proposals are here:
1. Fix of eslint issues
2. We would like to use webpack BEM loader, but we are not sure that we want to use [bem-react-core](https://github.com/bem/bem-react-core). So we get an error: `applyDecls` is `undefined`.
3. We have a few entry points in some projects with own levels of overrides. It looks crappy to execute a several builds with parameter for build all of project assets (one for every entry point).

### CODE REVIEW

**Comment by nikita-ezhkov** [comment]
@Yeti-or @awinogradov what do you think about this PR? Could it be merged?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```
Comment (Yeti-or) [inline .gitignore]: this sould be in your global git config
check https://git-scm.com/docs/gitignore section about excludesfile

File: generators/index.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-
-module.exports = {
+module.exports = (generators) => Object.assign({
```
Comment (Yeti-or) [inline generators/index.js]: no need of ()

**Comment by Yeti-or** [comment]
@nikita-ezhkov could you split this pr to 3 ones ?

**Comment by nikita-ezhkov** [comment]
@Yeti-or sure

**Pull request closed without merging**